### PR TITLE
Fixes #10 handle fieldtype settings

### LINKED ISSUE #10: The fieldType settings get lost when updating a contentType via API
tested by usage of the MigrationBundle: if a content class has an attribute of type ezmatrix, the names of the columns get reset when the class definition is updated

### PR DESCRIPTION
**Pull request by iherak**
Fixes https://github.com/ezcommunity/EzMatrixFieldTypeBundle/issues/10

### CODE REVIEW

**Comment by iherak** [comment]
@andrerom seems that the travis failed on php5.6 due to memory issue, not actual tests. Should I try to trigger the build again, or is this some known issue we can ignore?

**Comment by andrerom** [comment]
@iherak You can rebase PR, I updated kernel requirements in master and it seems to be able to install again now.

**Comment by iherak** [comment]
@andrerom I've had to upgrade phpunit version, but all tests seem to pass now.

File: Persistence/Legacy/Content/FieldValue/Converter/Matrix.php [hunk 1]
```diff
@@ -92,6 +93,13 @@ public function toStorageFieldDefinition( FieldDefinition $fieldDef, StorageFiel
      */
     public function toFieldDefinition( StorageFieldDefinition $storageDef, FieldDefinition $fieldDef )
     {
+        $fieldDef->fieldTypeConstraints->fieldSettings = new FieldSettings(
+            array
```
Comment (andrerom) [inline Persistence/Legacy/Content/FieldValue/Converter/Matrix.php]: Optionally: You can use `[]` here, but we can also fix that later by adding `php-cs-fixer` on the repo here so no need to jump on this unless you want.

Comment (iherak) [reply Persistence/Legacy/Content/FieldValue/Converter/Matrix.php]: I did not use that, as elsewhere in this class `array() ` is being used, so I thought it best to be consistent with the existing code.

**Comment by iherak** [comment]
@andrerom anything else blocking this from merging?

File: Persistence/Legacy/Content/FieldValue/Converter/Matrix.php (hunk 1 above)
Comment (oyvindlh) [reply Persistence/Legacy/Content/FieldValue/Converter/Matrix.php]: Any news on review? :)

**Comment by andrerom** [comment]
Let's go ahead with this, @gggeek if you see this please verify if it fixes your bug (you reported this).

**Pull request merged**